### PR TITLE
Added Tron and Championship Bowling 1600x1200 bezels

### PR DESCRIPTION
**Pull request by MDLAV8R**
First attempt here.  I think I managed to get everything uploaded correctly.  I will likely be adding more in the future as I stop modding and start actually playing my arcade machine :)

### CODE REVIEW

File: bezel-cfgs/pacplus.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Tempest 4:3 Overlay for Retropie 4.x - Retroarch
```
Comment (svera) [inline bezel-cfgs/pacplus.cfg]: This should be pacplus instead

**Review by svera — COMMENTED** [review]
In order to accept your contribution, could you please also add the 1024x768 version of the images? Thank you for your work!

**Comment by MDLAV8R** [comment]
Hi.  I create 1024x768 pngs and placed them in 1024 folder.  Also corrected the pacplus.cfg comment.

**Review by svera — APPROVED** [review]

**Comment by svera** [comment]
@MDLAV8R sorry, I decided to revert your contributions because images do not maintain the correct aspect ratio (you can see that images appear stretched). Could you please fix that and open a new pull request?

**Pull request merged**